src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs

```diff
@@ -43,19 +49,25 @@ public static void UseDefaultCredentials_Roundtrips()
             Assert.False(cws.Options.UseDefaultCredentials);
         }
 
-        [ActiveIssue(20358, TargetFrameworkMonikers.NetFramework)]
         [ConditionalFact(nameof(WebSocketsSupported))]
         public static void SetBuffer_InvalidArgs_Throws()
         {
             var cws = new ClientWebSocket();
             AssertExtensions.Throws<ArgumentOutOfRangeException>("receiveBufferSize", () => cws.Options.SetBuffer(0, 0));
-            AssertExtensions.Throws<ArgumentOutOfRangeException>("receiveBufferSize", () => cws.Options.SetBuffer(0, 1));
-            AssertExtensions.Throws<ArgumentOutOfRangeException>("sendBufferSize", () => cws.Options.SetBuffer(1, 0));
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("receiveBufferSize", () => cws.Options.SetBuffer(0, MinSendBufferSize));
```

Comment:
We should not modify the test case in order to make it pass on both .Net Core and Framework.

I think the better approach should be move the original test case to the `else {}` clause, and move the new one to `if (PlatformDetection.IsFullFramework)`